code/modules/unit_tests/limbsanity.dm

```diff
@@ -10,3 +10,28 @@
 				Fail("[path] does not have a valid icon for female variants")
 		else if(!icon_exists(UNLINT(part.should_draw_greyscale ? part.icon_greyscale : part.icon_static), "[part.limb_id]_[part.body_zone]"))
 			Fail("[path] does not have a valid icon")
+
+/datum/unit_test/limb_height_adjustment
+
+/// Tests the height adjustment system which dynamically changes how much the chest, head, and arms of a carbon are adjusted upwards or downwards based on the length of their legs and chest.
```

Comment:
```suggestion

/// Tests the height adjustment system which dynamically changes how much the chest, head, and arms of a carbon are adjusted upwards or downwards based on the length of their legs and chest.
/datum/unit_test/limb_height_adjustment

```

DMDocs was in the wrong spot, want it to be on the type definition, not on `Run()`.